iOS Client/PodcastsController/PodcastInfoTableViewCell.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  PodcastInfoTableViewCell.swift
+//  iOS Client
+//
+//  Created by Jonathan Witten on 8/4/18.
+//  Copyright © 2018 Mockingbird. All rights reserved.
+//
+
+import UIKit
+
+class PodcastInfoTableViewCell: UITableViewCell {
+
+
+    @IBOutlet weak var podcastName: UILabel!
+    @IBOutlet weak var podcastImage: UIImageView!
+    @IBOutlet weak var titleLabel: UILabel!
+    
+    func configure(image: UIImage, name: String, description: String) {
+        self.titleLabel.text = description
+        self.podcastImage.image = image
+        self.podcastName.text = name
+        self.titleLabel.font = UIFont(name: "IowanOldStyle-Roman", size: 18)
+        self.podcastName.font = UIFont(name: "IowanOldStyle-Roman", size: 14)
+        self.podcastName.textColor = UIColor.gray
+        self.selectionStyle = .none
```

Comment:
these bottom 4 rows don't need to be set every time configure is called. They can be moved up into an awakeFromNib function